### PR TITLE
Create table and Query Data 

### PR DESCRIPTION
**Pull request by AbbyNyakara**
Create the Animals database and query data. 
- The following are the queries and the subsequent results. 

-- 1.0 Find all animals whose name ends in "mon"
`SELECT * FROM animals WHERE name like '%mon';`

image
 
-- 2.0 List the name of all animals born between 2016 and 2019.
`SELECT name FROM animals WHERE YEAR(date_of_birth) BETWEEN '2016' AND '2019';`

image

-- 3.0 List the name of all animals that are neutered and have less than 3 escape attempts.
`SELECT name FROM animals WHERE neutered=TRUE and escape_attempts<3;`

image

-- 4.0 List the date of birth of all animals named either "Agumon" or "Pikachu".
`SELECT date_of_birth FROM animals WHERE name IN ("Agumon", "Pikachu");`

image

-- 5.0 List name and escape attempts of animals that weigh more than 10.5kg
`SELECT name, escape_attempts FROM animals WHERE weight_kg>10.5;`

image

-- 6.0 Find all animals that are neutered.
`SELECT * FROM animals WHERE neutered = true;`

image

-- -- 7.0 Find all animals not named Gabumon.
`SELECT * FROM animals WHERE name NOT IN ("Gabumon");`

image

-- -- 8.0 Find all animals with a weight between 10.4kg and 17.3kg (including the animals with the weights that equals precisely 10.4kg or 17.3kg)
`SELECT * FROM animals WHERE weight_kg >= 10.4 AND weight_kg <=17.3;`

image

### CODE REVIEW

File: schema.sql [hunk 1]
```diff
@@ -1,5 +1,14 @@
 /* Database schema to keep the structure of entire database. */
 
+CREATE DATABASE vet_clinic;
+
+USE vet_clinic;
+
 CREATE TABLE animals (
-    name varchar(100)
+    id INT NOT NULL PRIMARY KEY AUTO_INCREMENT,
```
Comment (brytebee) [inline schema.sql:8]: 👏👏👏👏 Awesome job making your `ID` field `NOT NULL` and `auto increment`.

File: data.sql [hunk 2]
```diff
@@ -1,5 +1,8 @@
...
 
-INSERT INTO animals (name) VALUES ('Luna');
-INSERT INTO animals (name) VALUES ('Daisy');
-INSERT INTO animals (name) VALUES ('Charlie');
+INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg)
+VALUES 
+("Agumon", "2020-02-03", 0, true, 10.23), 
+("Gabumon", "2018-11-15", 2, true, 8.0),
+("Pikachu", "2021-01-07", 1, false, 15.04),
+("Devimon", "2017-05-12", 5, false, 11.0);
```
Comment (brytebee) [inline data.sql]: - [x] **Required:** Kind update the neutered value of `Devimon` to true as required.

**Review by brytebee — CHANGES_REQUESTED** [review]
Hi, @AbbyNyakara👋,

Good job so far coming so far with the progress you've made! 💪💪

## Highlights
You have done amazing in the following:
- Created project using the given template.✔️
- Contains all the files required and has correct SQL commands in the right file.✔️
- Added screenshots of query results to the pull request description.✔️
 
However, there are some issues that you still need to work on to go to the next project but you are almost there!



# Required Changes ♻️
- Kindly update the title of your README to vet clinic or anything order than the curriculum name `curriculum-databases-projects-template.`
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Please, remember to tag me in your question so I can receive the notification.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @brytebee in your question so I can receive the notification.
### _**Use @brytebee on slack for faster responses.**_


**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: schema.sql (hunk 1 above)
Comment (AbbyNyakara) [reply schema.sql:8]: Thank you.

File: data.sql (hunk 2 above)
Comment (AbbyNyakara) [reply data.sql]: I have updated the data and query no. 6 that is affected

File: schema.sql (hunk 1 above)
Comment (AbbyNyakara) [reply schema.sql:8]: @brytebee , please review again?

**Review by zhadier — APPROVED** [review]
## Hi  @AbbyNyakara  👋  

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 



# Congratulations! 🎉 

✔️ No linter errors
✔️ Professional pull request title and description

### Keep Rocking!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**